packages/core/src/utils/addClusterStyle.ts

```diff
@@ -0,0 +1,28 @@
+import { MapConfig } from '@polar/lib-custom-types'
+import { Feature } from 'ol'
+import { InvisibleStyle, isVisible } from '@polar/lib-invisible-style'
+import { getDefaultStyle } from './markers'
+
+// optimization to keep getDefaultStyle memoization intact
+const defaultStyle = {}
+
+export const addClusterStyle = (mapConfiguration: MapConfig): MapConfig => {
+  mapConfiguration.layerConf.forEach((layerConfiguration) => {
+    if (layerConfiguration.clusterDistance !== undefined) {
+      const style =
+        mapConfiguration.extendedMasterportalapiMarkers?.defaultStyle ||
+        defaultStyle
+      // @masterportal/masterportalapi hook
+      layerConfiguration.style = (feature: Feature) => {
+        const visibleFeaturesCount = (feature.get('features') || []).filter(
```

Comment:
`feature.get('features')` doesn't have a type attached thus it is `any`, making `visibleFeaturesCount` `any` as well.

Easiest fix would be to just add `: number` behind `visibleFeaturesCount`

Reply:
https://github.com/Dataport/polar/pull/42/commits/16783312ff0efe6785e7ef1553077eb6668c5d6a Easiest fix applied.